from ._exceptions import DXPYTypeError